aries-site/src/pages/templates/wizard.mdx

```diff
@@ -73,6 +73,24 @@ If guidance can be provided in a short description, use a single column wizard.
 - On desktop, place the guidance to the right of form inputs.
 - On mobile, place the guidance above form inputs.
 
+## Alignment
```

Comment:
Would the anatomy treatment be a more digestible and consistent way for presenting this type of information?

Reply:
[Yes, agree](https://github.com/grommet/hpe-design-system/pull/2973#issuecomment-1281324168). I was trying to have this be a quick win last week by starting with some written guidance but since it carried over to this week I can add an anatomy diagram into this page.

Reply:
Done.